src/cascadia/TerminalConnection/ConhostConnection.cpp

```diff
@@ -33,6 +33,7 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
         _initialRows = initialRows;
         _initialCols = initialCols;
 
+        THROW_IF_FAILED(CoCreateGuid(&_guid));

```

Comment:
Somewhere else in the Cascadia code we're using `UuidCreate` -- now I'm not sure which one we should be using!

Reply:
CoCreateGuid calls UuidCreate under the covers, but has the added advantage that it returns an HRESULT intead of an RPC_STATUS and it takes a pointer to a GUID instead of a UUID.

From the docs:
["The CoCreateGuid function calls the RPC function UuidCreate, ..."](https://docs.microsoft.com/en-us/windows/desktop/api/combaseapi/nf-combaseapi-cocreateguid)

There was some other reason as well I ended up using this one instead in regards to header or library inclusion I hit, but that's why I chose CoCreateGuid.

If you'd like me to attempt to use UuuidCreate instead, I can, but wanted you to be aware of the tradeoffs.

Reply:
Further review indicates this is the function you're expected to call for WinRT apps as well since UuuidCreate is a Desktop API, so all the better to use it I think?

Reply:
totally. i'll switch the other use over in #913